### PR TITLE
Configure vocab for CJK

### LINKED ISSUE #745: Investigate issues with SentencePiece vocabulary for CJK
See comments from Jaume:

https://github.com/mozilla/firefox-translations-training/issues/45#issuecomment-1036191497
https://github.com/mozilla/firefox-translations-training/issues/45#issuecomment-1036198055

### PR DESCRIPTION
**Pull request by eu9ene**
- character coverage
- size

closes #745

### CODE REVIEW

File: pipeline/train/spm-vocab.sh [hunk 1]
```diff
@@ -70,6 +80,17 @@ zstdmt -dc "${merged_corpus_trg}" >"${vocab_dir}/data.trg.txt"
...
+      ${TRG} == 'zh' ||
+      ${SRC} == 'ja' ||
+      ${TRG} == 'ja' ||
+      ${SRC} == 'ko' ||
+      ${TRG} == 'ko' ]]
+then
+  character_coverage=0.9995
+else
+  character_coverage=1.0
+fi
```
Comment (ZJaume) [inline pipeline/train/spm-vocab.sh:93]: As I said in #745, I think default coverage would be enough for all languages if byte fallckback is enabled.

**Review by gregtatum — APPROVED** [review]
Seems reasonable to me.

File: pipeline/train/spm-vocab.sh (hunk 1 above)
Comment (eu9ene) [reply pipeline/train/spm-vocab.sh]: ok, replaced with sp default (removed the argument)

**Review by ZJaume — APPROVED** [review]

**Comment by ZJaume** [comment]
I'm thinking now that the 64K vocab as a workaround of split 32K vocabs may end up slowing down the decoding process because the softmax will be more expensive. Also I remember in the past the s2s models struggling to learn with large vocabulary outputs like 80k. So I wonder if we will see something of this in the Transformer. We'll see what happens in this preliminary experiments, but I'm starting to be more inclined to split vocabs.

**Comment by eu9ene** [comment]
> I'm thinking now that the 64K vocab as a workaround of split 32K vocabs may end up slowing down the decoding process because the softmax will be more expensive. Also I remember in the past the s2s models struggling to learn with large vocabulary outputs like 80k. So I wonder if we will see something of this in the Transformer. We'll see what happens in this preliminary experiments, but I'm starting to be more inclined to split vocabs.

yeah, I guess I'll rerun the experiment with the latest fixes and we can test it. I'll also work on https://github.com/mozilla/translations/issues/913 so that we can experiment with the split vocabs

**Pull request merged**